flight/Modules/AltitudeHold/altitudehold.c

```diff
@@ -153,6 +153,8 @@ static void altitudeHoldTask(void *parameters)
 	UAVObjEvent ev;
 	struct pid velocity_pid;
 
+	pid_zero(&velocity_pid);
```

Comment:
velocity_pid's antiwindup term could be uninit'd without this change-- potentially the most serious one.